src/modules/logger/logger.cpp

```diff
@@ -510,8 +510,8 @@ void Logger::add_default_topics()
 {
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
-	add_topic("airspeed", 200);
```

Comment:
I didn't go through all topics, but what is driving this reduction? Where is the same data available at higher rate?

Reply:
It's included at full rate with the "Estimator replay" profile, which is selected by default. Then in any actual controller usage we have it in `tecs_status` as well.